book/01-introduction/sections/basics.asc

```diff
@@ -34,7 +34,8 @@ Most operations in Git only need local files and resources to operate – genera
 If you're used to a CVCS where most operations have that network latency overhead, this aspect of Git will make you think that the gods of speed have blessed Git with unworldly powers.
 Because you have the entire history of the project right there on your local disk, most operations seem almost instantaneous.
 
-For example, to browse the history of the project, Git doesn't need to go out to the server to get the history and display it for you – it simply reads it directly from your local database.
+For example, to browse the history of the project, Git doesn't need to go out to the server to get the history and display it for you – it simply reads it directly from your local database
+(commonly called the _object store_).
```

Comment:
First: the convention in this repo is to keep all sentences on a single line, and let your editor soft-wrap. That makes diffing easier – notice how it didn't highlight the deleted period on the first line here. 

Second: this is chapter 1. The number of technical terms we want to introduce here is as close to zero as possible, and the concept that a thing can have two names doesn't fit here.

Third: it's actually called the "object database."

Could you roll back this change?